### DISCUSSION TITLE
Interactive SVG documents

### DISCUSSION THREAD

**Discussion by 3goats** [squidfunk/mkdocs-material#6014] [General]
Hi,

I have an SVG document that includes some javascript that adds some interactive capabilities. This works fine if I open it directly in a browser. However, if I add it to my docs the interactivity does not work. I assume because it's not inline. I then tried adding the `mkdocs-plugin-inline-svg` however that doesn't seem to be supported by mkdocs-material. 

Is there a preferred way to to add interactive SVG or make my SVG interactive in mkdocs-material ?

**Comment by alexvoss** [comment]
Hi,

I have a personal interest to get interactive SVGs working, so may want to look at this. When you say that it does not seem that `mkdocs-plugin-inline-svg` is supported by mkdocs-material, what do you go by? 

Cheers, Alex

**Comment by 3goats** [comment]
When I try to add the plugin to my mkdocs.yaml using VSCode, it flags an error: 

`Value is not accepted. Valid values: "blog", "info", "meta", "offline", "optimize", "privacy", "projects", "search", "social", "tags", "typeset", "git-authors", "git-committers", "git-revision-date", "literate-nav", "macros", "section-index", "include-markdown", "material-relative-language-selector", "rss", "git-revision-date-localized", "glightbox", "meta-descriptions", "mkdocstrings", "coverage", "spellcheck", "markdown-exec".yaml-schema: Built-in blog plugin(1)`

Also, it seems that the plugin is then just ignored - I think. 

Carl

**Reply by alexvoss** [reply]
Interesting, this is VS Code doing the validation against a schema, right? Did you set this up at some point? If so, what happens when you deactivate the validation (you should see the schema at the bottom right of the screen, I think). I am working on another issue at the moment but will try to try out the plugin myself to see if it works.

**Reply by lampensau** [reply]
The plugin `mkdocs-plugin-inline-svg` is working fine for me. Sure, it is not included in the [VS Code schema](https://squidfunk.github.io/mkdocs-material/schema.json), but that is not impeding the plugin's function for me.

Did you check the generated HTML after performing `mkdocs build` or `mkdocs serve`?

**Reply by alexvoss** [reply]
@lampensau - thanks for confirming this. 

@3goats can you let me know if the schema is the only problem? If not then it would be good if you could produce a [minimal repoduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) so I can run your specific code and see where the problem might lie.

**Comment by 3goats** [comment]
OK that's working now. I used the wrong value for enabling the plug-in. Thanks for your help.
